src/components/Products/ProductForm.jsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from "react";
+import { addProduct, updateProduct } from "../../api/productService";
+
+const ProductForm = ({ product, onProductAdded, onProductUpdated }) => {
+	const [name, setName] = useState(product ? product.name : "");
+	const [type, setType] = useState(product ? product.type : "");
+	const [description, setDescription] = useState(
+		product ? product.description : ""
+	);
+	const [photo, setPhoto] = useState(null);  // Set initial state to null for the file
+
+	const handlePhotoChange = (e) => {
+		setPhoto(e.target.files[0]);  // Store the uploaded file in state
```

Comment:
are we handing the case where user add image on frontend and removal it before uploading it to server?